src/rules/function-scale-color-use-always/index.js

```diff
@@ -0,0 +1,52 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+import valueParser from "postcss-value-parser";
+
+export const ruleName = namespace("function-scale-color-use-always");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Use the scale-color function."
```

Comment:
As I pointed out in the other reviews, the naming convention in stylelint error messages is `"Expected [something] [in some context]"` and `"Unexpected [something] [in some context]"`. We should try to stick to that message format in the future.

https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#determine-violation-messages